.travis.yml

```diff
@@ -51,7 +51,7 @@ after_script:
 
 branches:
   only:
-    - master
+    - /.*/
```

Comment:
What does this mean?
(Would be great if we add a comment before this line)

Reply:
It's a regular expression to build all branches.

Reply:
Got it!

Reply:
Got it!